src/aat/resources/features/F-131 - AC CRUD createEvent external/S-131.1.td.json

```diff
@@ -0,0 +1,42 @@
+{
+  "_guid_" : "S-131.1",
+  "_extends_" : "F-131_Test_Data_Base",
+
+  "specs" : [
+    "with no C access to create an event",
+    "attempts to create an event for the previously created case"
+  ],
+
+  "request": {
+    "_extends_": "Common_Request",
+    "pathVariables": {
+      "caseId": "${[scenarioContext][childContexts][F-131_CreateCase][testData][actualResponse][body][id]}"
+    },
+
+    "headers": {
+      "experimental": true
+    },
+    "body": {
+      "data": {
+        "TextField4" : "value4"
+      },
+      "event": {
+        "id" : "event1"
+      },
+      "ignore_warning": false,
```

Comment:
we can remove this line as it's in the base file

Reply:
Have removed this line now